style.css

```diff
@@ -97,4 +116,328 @@ ul {
+    display: flex;
+    flex-flow: row;
+    width: 95%;
+    margin-top: 1%;
+    margin-bottom: 4%;
+    margin-left: -8%;
+  }
+
+  .first_box_lang li {
+    background-color: transparent;
+    color: black;
+    width: fit-content;
+    border: 1px solid black;
+    padding: 10px;
+    margin: 0 11px 0 0;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .first_box_button {
+    font-family: 'Inter', sans-serif;
+    font-weight: bold;
+    font-size: 17px;
+    margin-right: auto;
+    margin-top: -1%;
+    background: #ff6b00;
+    width: 34%;
+    height: 8%;
+    color: white;
+    margin-left: 6%;
+    border: 0;
+    display: inline-block;
+    cursor: pointer;
+  }
+
+  /* Hover state */
+  .first_box_button:hover {
+    color: #ff6b00;
+    background-color: antiquewhite;
+    border: 1px solid #ff6b00;
+  }
+
+  /* Pressed state */
+  .first_box_button:active {
+    background: #c65504;
+    color: white;
+  }
+
+  /* Disabled state */
+  .first_box_button:disabled {
+    background-color: #ccc;
+    cursor: not-allowed;
+  }
+
+  .box {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    word-wrap: break-word;
+    width: 20.438rem;
+    height: 24.125rem;
+    margin: 0 0 0 0;
+    background-image: url(resources/bg.png);
+    background-size: cover;
+    color: white;
+    padding-top: 10%;
+  }
+
+  .box h2 {
+    font-size: 32px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 44px;
+    margin: 16% 0 0 5%;
+    color: white;
+  }
+
+  .box p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: white;
+    width: 88%;
+    padding-left: 5%;
+  }
+
+  .lang_used {
+    display: flex;
+    flex-flow: row;
+    padding: 0%;
+    margin: 9px;
+  }
+
+  .lang_used li {
+    font-family: 'Inter', sans-serif;
+    background-color: rgba(255, 255, 255, 0.201);
+    color: white;
+    padding: 10px;
+    margin: 0 0 5px 8px;
+  }
+
+  .box button {
+    margin-top: auto;
+    font-family: 'Inter', sans-serif;
+    font-weight: bold;
+    font-size: 17px;
+    background: #ff6b00;
+    width: 100%;
+    height: 3rem;
+    color: white;
+    border: 0;
+  }
+
+  /* Hover state */
+  .box button:hover {
+    color: #ff6b00;
+    background-color: antiquewhite;
+    border: 1px solid #ff6b00;
+  }
+
+  /* Pressed state */
+  .box button:active {
+    background: #c65504;
+    color: white;
+  }
+
+  /* Disabled state */
+  .box button:disabled {
+    background-color: #ccc;
+    cursor: not-allowed;
+  }
+
+  /* about section */
+  .about {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    background-color: #1c1a19;
+    height: 100%;
+    row-gap: 1.625rem;
+    margin-bottom: 0;
+  }
+
+  .about_section {
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    height: 26.188rem;
+    margin: 4.75rem 0 0 1.188rem;
+  }
```

Comment:
- [ ] As stated in the project requirement Make sure the content of the "About myself" section is aligned with `flexbox. `So for the about section kindly update the section aligned with `flexxbox `instead of the `grid.`

Reply:
@Abenezer-Tilahun Done!